### PR TITLE
Gtk: Fix input of gui-console for characters & and < by escaping them since rich-text is XML based

### PR DESCRIPTION
**Pull request by rcqls**
Add: read-clipboard and write-clipboard completed for Linux based on gtk_clipboard.

### CODE REVIEW

File: modules/view/backends/gtk3/gtk.reds [hunk 1]
```diff
@@ -662,6 +667,37 @@ GPtrArray!: alias struct! [
 			display 	[handle!]
 			return: 	[handle!]
 		]
+		gtk_clipboard_get: "gtk_clipboard_get" [
+			atom 		[handle!]
+			return: 	[handle!]
+		]
+		gtk_clipboard_set_text: "gtk_clipboard_set_text" [
```
Comment (qtxie) [inline modules/view/backends/gtk3/gtk.reds]: Is it used in GTK backend? It's imported in clipboard.reds too.

Comment (rcqls) [reply modules/view/backends/gtk3/gtk.reds]: It is put in both gtk.reds and clipboard.reds

Comment (qtxie) [reply modules/view/backends/gtk3/gtk.reds]: Why? if it's only needed in clipboard.reds, import it once is enough.

Comment (rcqls) [reply modules/view/backends/gtk3/gtk.reds]: I put it first only in gtk.reds (which is IIMHO, the right file) but it did not compile. I added then in clipboard.reds. Also I used the gtk.reds as is as a gtk-plugins for red/system.

**Comment by rcqls** [comment]
Added to the PR some fix about visible issue for tips.

**Comment by bitbegin** [comment]
https://www.uninformativ.de/blog/postings/2017-04-02/0/POSTING-en.html
we can use x windows clipboard

**Comment by rcqls** [comment]
@bitbegin Tha's was also an option. I checked `libxcb.so` but since we already use the GTK backend, I thought it was easier to directly use it.

**Comment by bitbegin** [comment]
https://gist.github.com/bluecube/9835979 
yes, this's very complicated。

File: modules/view/backends/gtk3/gtk.reds (hunk 1 above)
Comment (qtxie) [reply modules/view/backends/gtk3/gtk.reds]: After those functions are imported in clipboard.reds, you can use them in the GTK backend. Just add the context prefix when calling it. e.g. `clipboard/gtk_clipboard_set_text ...`.

Comment (rcqls) [reply modules/view/backends/gtk3/gtk.reds]: Thanks for this explanation!

**Pull request merged**